tests/specs/browser.form.spec.ts

```diff
@@ -0,0 +1,17 @@
+import FormPage from '../pageobjects/form.page';
+
+fdescribe('auth form', () => {
```

Comment:
was this intentional?
```suggestion
describe('auth form', () => {
```

Reply:
Thanks, was not intentional